cmd/kyma/apply/function/opts.go

```diff
@@ -0,0 +1,48 @@
+package function
+
+import (
+	"github.com/kyma-project/cli/internal/cli"
+)
+
+//Options defines available options for the command
+type Options struct {
+	*cli.Options
+
+	OnError  OnError
+	Output   Output
+	Filename string
+	DryRun   bool
+}
+
+type OnError = string
```

Comment:
```suggestion
type OnError string
```

Reply:
Why? This is only an alias to the string, to clarify the code

Reply:
Ah I see... So it is not your intention to create a new string type?
I assumed it was that since you have fields and constants with that type.